lib/r10k/api/util.rb

```diff
@@ -31,6 +31,14 @@ def cachedir_for_forge_module(module_slug, cachedir=nil)
         return File.join(cachedir, 'forge', module_slug)
       end
 
+      def module_slug_from_release_slug(release_slug)
```

Comment:
This is not the only place where we need to handle /'s. RK-225 is created to asses and fix the slash-dash issue.